scripts/detect_missing_i18n.py

```diff
@@ -197,7 +181,7 @@ class Errtype(str, Enum):
     + ignore_single_quote
     + r")[^>]*?>"
 )
-i18n_attr_warn_regex = opening_tag_open + attr_syntax + r"['\"]\$\((?!_\()"
+i18n_attr_warn_regex = opening_tag_open + attr_syntax + r"\"\$\(\'"
```

Comment:
Ditto here

Reply:
Same thing as above, except in this case since it will always be nested in double quotes, i.e. `title=""`, we're just matching `$('`.

But could easily use the same regex both places and put it in a new variable for clarity! Probably the wisest course of action